auto-cli.bat

```diff
@@ -252,7 +252,7 @@ if exist ..\revanced-cli-input\patches.jar (
 	pause
 	"%JDK%" -jar "%CLI%" list-patches -dopv "%PATCHES%"
 ) else (
-	echo  You are using official ReVanced patches. Please look up the patches names at revanced.app/patches.
+	echo  You are using official ReVanced patches. [93m Please look up the patch names and capitalizations at https://revanced.app/patches.[0m
```

Comment:
i hope i highlighted correctly.

i honestly think this should be stressed more, or somehow placed in the paragraph below